repos/system_upgrade/el8toel9/actors/nisscanner/tests/test_nisscan.py

```diff
@@ -0,0 +1,83 @@
+import mock
+import pytest
+import six
+
+from leapp.libraries.actor import nisscan
+from leapp.libraries.common.testutils import CurrentActorMocked, produce_mocked
+from leapp.libraries.stdlib import api
+from leapp.models import NISConfig
+
+
+@pytest.mark.parametrize('pkgs_installed', [
+    ('ypserv',),
+    ('ypbind',),
+    ('ypserv-not-ypbind',),
+    ('ypbind', 'ypserv'),
+])
+@pytest.mark.parametrize("fill_conf_file", [True, False])
+@pytest.mark.parametrize("fill_ypserv_dir", [True, False])
+def test_actor_nisscan(monkeypatch, pkgs_installed, fill_conf_file, fill_ypserv_dir):
+    """
+    Parametrized helper function for test_actor_* functions.
+
+    Run the actor feeded with our mocked functions and assert
+    produced messages according to set arguments.
+
+    Parameters:
+        pkgs_installed  (touple): installed pkgs
+        fill_conf_file  (bool): not default ypbind config file
+        fill_ypserv_dir  (bool): not default ypserv dir content
+    """
+
+    # Examples of /etc/yp.conf configuration file (default and configured)
+    YPBIND_DEFAULT_CONF = """
+    # /etc/yp.conf - ypbind configuration file
```

Comment:
I believe you should remove whitespaces at the beginning of L34-39 ans L42-46

Reply:
Yes, sorry. I use join when I need to create multi line string because it is more readable most of the time.